contracts/azorius/interfaces/IVersion.sol

```diff
@@ -0,0 +1,13 @@
+//SPDX-License-Identifier: MIT
+pragma solidity =0.8.19;
+
+/**
+ * Interface of functions for versioned contracts
+ */
+interface IVersion {
+    /**
+     * Returns the current version of the contract
+     *
+     */
+    function getVersion() external pure returns (uint8);
```

Comment:
Maybe we actually do want this type to be larger? I guess there could be a world in which we exceed 256 versions.